internal/store/pod.go

```diff
@@ -1568,6 +1569,31 @@ func createPodStatusUnschedulableFamilyGenerator() generator.FamilyGenerator {
 	)
 }
 
+func createPodReadyTimeFamilyGeneratorFamilyGenerator() generator.FamilyGenerator {
```

Comment:
**nitpick;** I'd suggest renaming it to `createPodStatusReadyTimeFamilyGenerator` to keep it consistent with others such as `createPodStatusScheduledTimeFamilyGenerator`.